tests/sentry/db/models/fields/bitfield/test_bitfield.py

```diff
@@ -331,16 +338,12 @@ def test_dictionary_init(self):
         pytest.raises(ValueError, BitField, flags={"1": "non_int_key"})
 
     def test_defaults_as_key_names(self):
-        class TestModel(models.Model):
```

Comment:
Moved this out of this test function primarily since this model didn't seem to be cleaned up properly after running the test, leading to the failure of new backup/restore tests. It now also matches how the rest of the test bitfield models are set up